governance/votes/Executive Vote - October 11, 2023.md

```diff
@@ -0,0 +1,195 @@
+| Harmony         | [0xE20A2e231215e9b7Aa308463F1A7490b2ECE55D3](https://etherscan.io/address/0xE20A2e231215e9b7Aa308463F1A7490b2ECE55D3) | Resiliency           | 20.85        |
+| Res             | [0x8c5c8d76372954922400e4654AF7694e158AB784](https://etherscan.io/address/0x8c5c8d76372954922400e4654AF7694e158AB784) | Resiliency           | 20.85        |
+| seedlatam.eth   | [0x0087a081a9b430fd8f688c6ac5dd24421bfb060d](https://etherscan.io/address/0x0087a081a9b430fd8f688c6ac5dd24421bfb060d) | Sovereign Finance    | 20.85        |
+
+### Aligned Delegate (AD) Member Monthly Compensation
+
+*Authorization*: [Atlas (MIP101)](https://mips.makerdao.com/mips/details/MIP101#2-6-3-aligned-delegate-ad-income-and-participation-requirements)\
+*Proposal*: [Forum post](https://forum.makerdao.com/t/september-2023-aligned-delegate-compensation/22367) "September 2023 Aligned Delegate Compensation"
+
+If this executive proposal passes, the following distributions will be made to qualified AD members for September Compensation, totaling **269.28 MKR**:
+
+| Delegate    | Address                                                                                              | Amount (MKR) |
+|-------------|------------------------------------------------------------------------------------------------------|--------|
+| 0xDefensor  | [0x9542b441d65B6BF4dDdd3d4D2a66D8dCB9EE07a9](https://etherscan.io/address/0x9542b441d65B6BF4dDdd3d4D2a66D8dCB9EE07a9) | 41.67  |
+| TRUE NAME   | [0x612f7924c367575a0edf21333d96b15f1b345a5d](https://etherscan.io/address/0x612f7924c367575a0edf21333d96b15f1b345a5d) | 41.67  |
+| BONAPUBLICA | [0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3](https://etherscan.io/address/0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3) | 41.67  |
+| Navigator   | [0x11406a9CC2e37425F15f920F494A51133ac93072](https://etherscan.io/address/0x11406a9CC2e37425F15f920F494A51133ac93072) | 41.67  |
+| vigilant    | [0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61](https://etherscan.io/address/0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61) | 34.5   |
+| Cloaky      | [0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818](https://etherscan.io/address/0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818) | 21.06  |
+| UPMaker     | [0xbb819df169670dc71a16f58f55956fe642cc6bcd](https://etherscan.io/address/0xbb819df169670dc71a16f58f55956fe642cc6bcd) | 13.89  |
+| PALC        | [0x78Deac4F87BD8007b9cb56B8d53889ed5374e83A](https://etherscan.io/address/0x78Deac4F87BD8007b9cb56B8d53889ed5374e83A) | 13.89  |
+| BLUE        | [0xb6c09680d822f162449cdfb8248a7d3fc26ec9bf](https://etherscan.io/address/0xb6c09680d822f162449cdfb8248a7d3fc26ec9bf) | 12.78  |
+| PBG         | [0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2](https://etherscan.io/address/0x8D4df847dB7FfE0B46AF084fE031F7691C6478c2) | 6.48   |
+
+
+### Governance Facilitator Payment Streams
+
+*Authorization*: [Governance Poll](https://vote.makerdao.com/polling/QmSovaxn)\
```

Comment:
Shouldn't we add this MIP113 reference instead of the ratification poll?
https://mips.makerdao.com/mips/details/MIP113#7-1-1-1-1a

Reply:
I'm in favour of this. In the past, Executives often referenced MKR holder approval, but I think this is a more reasonable direction.